<?xml version="1.0" encoding="UTF-8"?><testsuite errors="1" failures="1" name="com.studentapp.junit.studentsinfo.FirstSerenityTest" skipped="4" tests="8" time="3.11" timestamp="2021-02-04 03:38:35"><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="This is a skipped test"><skipped/></testcase><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="This is atest with error"><error message="/ by zero" type="java.lang.ArithmeticException">/ by zero</error><system-err>/ by zero&#13;
com.studentapp.junit.studentsinfo.FirstSerenityTest.thisIsAtestWithError(FirstSerenityTest.java:62)&#13;
</system-err></testcase><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="This test will get the information of all the students from the Student App"/><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="This is a pending test"><skipped/></testcase><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="This is a manual test"><skipped/></testcase><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="This isa failing"><failure message="1 expectation failed.&#10;Expected status code &lt;500&gt; but was &lt;200&gt;.&#10;" type="java.lang.AssertionError">1 expectation failed.
Expected status code &lt;500&gt; but was &lt;200&gt;.
</failure><system-err>1 expectation failed.
Expected status code &lt;500&gt; but was &lt;200&gt;.
&#13;
io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:674)&#13;
io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:126)&#13;
io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:134)&#13;
io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)&#13;
io.restassured.internal.ValidatableResponseImpl.super$2$statusCode(ValidatableResponseImpl.groovy)&#13;
groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)&#13;
groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1267)&#13;
io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy:142)&#13;
io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy)&#13;
com.studentapp.junit.studentsinfo.FirstSerenityTest.thisIsaFailing(FirstSerenityTest.java:44)&#13;
</system-err></testcase><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="File does not exist"><compromised message="E:\file.txt (The system cannot find the path specified)" type="java.io.FileNotFoundException">E:\file.txt (The system cannot find the path specified)</compromised><system-err>E:\file.txt (The system cannot find the path specified)&#13;
com.studentapp.junit.studentsinfo.FirstSerenityTest.fileDoesNotExist(FirstSerenityTest.java:69)&#13;
</system-err></testcase><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="Get all students"/></testsuite>